packages/modeling/src/operations/transforms/scale.js

```diff
@@ -15,12 +15,16 @@ const { geom2, geom3, path2 } = require('../../geometry')
  */
 const scale = (factors, ...objects) => {
   if (!Array.isArray(factors)) throw new Error('factors must be an array')
-  if (factors.length !== 3) throw new Error('factors must contain X, Y and Z values')
-  if (factors[0] <= 0 || factors[1] <= 0 || factors[2] <= 0) throw new Error('factors must be positive')
 
   objects = flatten(objects)
   if (objects.length === 0) throw new Error('wrong number of arguments')
 
+  // adjust the factors if necessary
+  factors = factors.slice() // don't modify the original
```

Comment:
same as with rotate about using toArray() 
In this case it would also make it closer to V1 api and allow scale(0.5, objects), which is very useful